lib/exceptions/backends/rollbar.rb

```diff
@@ -0,0 +1,50 @@
+require 'rollbar'
+
+module Exceptions
+  module Backends
+    # Public: The Rollbar backend is a Backend implementation that sends the
+    # exception to Rollbar.
+    class Rollbar
+      DEFAULT_NOTIFY_ARGS = {
+        # without this, rollbar won't ignore the error classes defined in
+        # config/initializers/exceptions.rb
+        use_exception_level_filters: true
+      }
+
+      attr_reader :rollbar
+
+      def initialize(rollbar = ::Rollbar)
+        @rollbar = rollbar
+      end
+
+      def notify(exception = nil, level: 'error', error_class: nil, error_message: nil, parameters: {}, context: {})
+        err = exception || error_class
+        extra = [DEFAULT_NOTIFY_ARGS, parameters, context].reduce(&:merge)
+        Response.wrap(rollbar.log(level, err, error_message, extra))
+      end
+
+      def context(ctx)
+        # rollbar does special tracking of the `person` key, but `user` is a more
+        # common name for this.
+        ctx = Marshal.load(Marshal.dump(ctx)) # deep clone
```

Comment:
I just removed this deep clone hack since shallow dup is enough for the thing I'm trying to achieve